hardhat.config.js

```diff
@@ -27,13 +27,24 @@ const optimizerSettings = {
         },
     },
 };
+const itsOptimizerSettings = {
+    ...optimizerSettings,
+    runs: 600, // Reduce runs to keep bytecode size under limit
+};
 const compilerSettings = {
     version: '0.8.21',
     settings: {
         evmVersion: process.env.EVM_VERSION || 'london',
         optimizer: optimizerSettings,
     },
 };
+const itsCompilerSettings = {
+    version: '0.8.21',
+    settings: {
+        evmVersion: process.env.EVM_VERSION || 'london',
+        optimizer: itsOptimizerSettings,
```

Comment:
```suggestion
        optimizer: {
            ...optimizerSettings,
            runs: 600, // Reduce runs to keep bytecode size under limit
        },
```
a bit more concise maybe